common/components/register_model/spec.yaml

```diff
@@ -17,9 +17,11 @@ command: >-
   --model_path ${{inputs.model_path}}
   $[[--model_type ${{inputs.model_type}}]]
   $[[--model_name ${{inputs.model_name}}]]
+  $[[--model_version ${{inputs.model_version}}]]
   $[[--registry_name ${{inputs.registry_name}}]]
-  $[[--model_info_path ${{inputs.model_info_path}}]]
+  $[[--download_details ${{inputs.download_details}}]]
```

Comment:
@ManojBableshwar suggested to go with `model_download_metadata`. Can you please modify the naming here?

Reply:
Done